file1.txt

```diff
@@ -3,4 +3,5 @@ commit 1 b1 git2
 commit 2 b2 git2  
 commit 3 b2 git2
 commit 1 b3 git2 
+commit 2 b1 git2 
```

Comment:
I will check.  

    On Wednesday, May 4, 2016 12:12 PM, razvaniuoras <notifications@github.com> wrote:
 

 In file1.txt:> @@ -3,4 +3,5 @@ commit 1 b1 git2
>  commit 2 b2 git2  
>  commit 3 b2 git2
>  commit 1 b3 git2 
> +commit 2 b1 git2 
New commit b1. —
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub